proto/common/healthcheck/healthcheck.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package conduit.common.healthcheck;
+
+enum CheckStatus {
+    OK = 0;
+    FAIL = 1;
+    ERROR = 2;
+}
+
+message CheckResult {
+    string SubsystemName = 1;
+    string CheckDescription = 2;
+    CheckStatus Status = 3;
+    string FriendlyMessageToUser = 4;
+}
+
+message Check {
+    repeated CheckResult results = 1;
+    CheckStatus overallStatus = 2;
```

Comment:
realize this was already implemented this way, but is `overallStatus` redundant? can it just be inferred from evaluating all elements in `CheckResult`?

Reply:
Cool, yeah, all makes sense. I've removed this message from the protobuf definition.

I also realized the `Check` struct was only used as return value for the `PerformCheck` method in healthcheck.go, and the only call site for that method wasn't reading the value of the results field. So I've simplified the `PerformCheck` method to return `pb.CheckStatus`, and removed this struct entirely.